pages/roadmap/[...slug].tsx

```diff
@@ -94,12 +94,14 @@ export default function RoadmapPage(props: InferGetServerSidePropsType<typeof ge
 
     fetchRoadMap();
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [issue_number, repo, owner]);
 
   useEffect(() => {
     if (isPendingChildrenLoading) return;
     const typedPendingChild = pendingChildrenState[0];
     if (typedPendingChild == null || typedPendingChild.html_url?.value == null) {
+      pendingChildrenState[0].set(none);
```

Comment:
this resolves a large part of our rendering issues. pendingChildrenState was set to [null, null] for some reason, and wasn't being cleared out. `@hookstate/core` will try to always append for objects/arrays unless you set via the callback method, so it was getting filled with invalid items and not being cleared out.